app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
     defaultConfig {
         applicationId "com.odysee.app"
         minSdkVersion 21
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
This change is not needed and makes Gradle warn about it. It won't be hurting the app to use Android API Level 30